src/js/components/AppPageComponent.jsx

```diff
@@ -157,7 +157,16 @@ var AppPageComponent = React.createClass({
     });
   },
 
-  onScaleAppError: function (errorMessage) {
+  onScaleAppError: function (errorMessage, statusCode, instances) {
+    if (statusCode === 409) {
+      var appId = this.state.appId;
+      const dialogId = DialogActions.
+        confirm("App scaling failed.Do you want to try forcing?");
```

Comment:
So the new message you are proposing is something like this:
> Failed to scale ${appId}. Do you want to try forcing the scale action? This will stop any current deployment of the app, and start a new one to scale it.